pkg/crypto/crypto.go

```diff
@@ -72,6 +80,14 @@ func TLSVersionOrDie(versionName string) uint16 {
 	}
 	return version
 }
+func SupportedTLSVersions() []string {
```

Comment:
Clarify the difference between supported and valid for me. I don't understand it.